fuzz/fuzz_targets/value_binop.rs

```diff
@@ -15,13 +19,111 @@ enum BinOp {
     Cmp,
 }
 
-#[derive(Debug, arbitrary::Arbitrary)]
+#[derive(Debug)]
 struct Input {
     op: BinOp,
     lhs: Value,
     rhs: Value,
 }
 
+::std::thread_local! {# [allow (non_upper_case_globals )]static RECURSIVE_COUNT_Value : core::cell::Cell<u32> = const {core::cell::Cell::new(0)}; }
```

Comment:
Not opposed to this approach, especially since it's fuzzing infra only, but would it be simpler to pass the current depth around in `arbitrary_value`?

```rust
fn arbitrary_value(u: &mut arbitrary::Unstructured<'_>, depth: usize) -> arbitrary::Result<Value> {
    // ...
    arbitrary_value(..., depth + 1);
}
```